Create shop

paluszkiewiczB opened the pull request:
Zachęcam do komentowania, subskrybowania i lajkowania.
closes #22

paluszkiewiczB committed:
signing up as owner requires creating shop;
extract zip code to separate class;

Commit:
add shop creation as part of signup process;
cover ShopService with unit test;
implement ShopValidator;
add open hours validation;

jkkostrzewski commented on src/main/java/com/zachaczcompany/zzpj/security/jwt/UserRegistrationService.java:
```diff
@@ -1,51 +1,67 @@
     }
 
+    private static Response registerAndAssignToShop(UserSignUp userSignUp) {
+        throw new IllegalStateException("NOT IMPLEMENTED YET");
```
zostaw todosa byczq

jkkostrzewski reviewed (approved):
pushowałbym jak ruscy b

Commit:
add todo;

Commit:
Merge remote-tracking branch 'origin/master' into createShop

# Conflicts:
#	src/main/java/com/zachaczcompany/zzpj/commons/response/Error.java
#	src/main/java/com/zachaczcompany/zzpj/shops/domain/ShopFacade.java
#	src/main/java/com/zachaczcompany/zzpj/shops/domain/ShopService.java
#	src/test/groovy/com/zachaczcompany/zzpj/shops/domain/ShopTestsDataProvider.groovy

Commit:
resolve conflicts;
mv business logic and validation to service and validator accordingly;
change misleading variable names;
add static imports to tests;

Commit:
Merge remote-tracking branch 'origin/master' into createShop

Commit:
fix test;
